components/docs-chef-io/content/automate/reusable/md/restore_troubleshooting.md

```diff
@@ -19,3 +19,173 @@ Try these steps if Chef Automate returns an error while restoring data.
     ```
 
 Now check the status of the Automate node and then try running the restore command from the bastion host.
+
+1. How to change the `base_path` or `path`. Below steps for the File System backup
+   - While at the time of deployment `backup_mount` default value will be `/mnt/automate_backups`
+   - In case, if you modify the `backup_mount` in `config.toml` before deploment, then deployment process will do the configuration with updated value
```

Comment:
- In case, if you modify the `backup_mount` in `config.toml` before deployment, then the deployment process will do the configuration with the updated value